api/pkg/import/zip_package.go

```diff
@@ -0,0 +1,149 @@
+package _import
+
+import (
+	"archive/zip"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+// ErrorUncompressedSizeTooBig signals that a zip archive has an uncompressed size greater than maximum size allowed
+var /*const*/ ErrorUncompressedSizeTooBig = errors.New("uncompressed size of package exceeds configured maximum size")
```

Comment:
```suggestion
const ErrorUncompressedSizeTooBig = errors.New("uncompressed size of package exceeds configured maximum size")
```

Reply:
this cannot be done as the right-hand-side expression is not a constant expression.
We have to keep the constant as a global variable in this case